docs/dev/tools/cli.md

```diff
@@ -691,7 +691,7 @@ viam machines list
 viam machines status  --machine=123
 
 # create an api key for a machine
-viam machines api-key create --machine=123 --name=MyKey
+viam machines api-key create --machine-id=123 --org-id=456 --name=MyKey
```

Comment:
if you use machine-id you don't need org-id too though do you? since machine-id is unique?